src/Components/ServiceScene/ServiceScene.tsx

```diff
@@ -444,7 +444,7 @@ const MAIN_PANEL_MAX_HEIGHT = '30%';
 function buildGraphScene() {
   return new SceneFlexLayout({
     direction: 'column',
-    $behaviors: [new behaviors.CursorSync({ key: 'metricCrosshairSync', sync: DashboardCursorSync.Crosshair })],
+    $behaviors: [new behaviors.CursorSync({ key: 'logsCrosshairSync', sync: DashboardCursorSync.Crosshair })],
```

Comment:
better name, but what are we trying to sync since this is only a single panel?

Reply:
It has no effect. Removing it causes the same bug we see in the service selection page, so what I would do is to also set it as `DashboardCursorSync.None`.

This comes from the hackathon: https://github.com/grafana/hackthon-2024-03-logger-info/blob/main/src/components/Explore/LogsByService/LogsByServiceScene.tsx#L349

@adrapereira Do you remember if we have any reason to keep this?

Reply:
I'd set it to none, or figure out if we can sync crossHairs between logs volume and the logs panel (but in another issue/PR), but I'm not sure that would work since the buckets from the aggregation wouldn't line up with the timestamps on the logs?

Reply:
Agree. I'll do it in a follow-up because, for some reason, setting it to `Off` somehow enables it.

Reply:
Ahhh good point. I never payed attention to it, I'll see if that actually ever happens. Thank you!

Reply:
Yep, it's adding a crosshair between Logs Volume and the other panels. Keeping it then, thank you again @adrapereira